src/mcts/search.cc

```diff
@@ -713,13 +712,13 @@ int SearchWorker::PrefetchIntoCache(Node* node, int budget) {
     Node* n = scores[i].second;
     // Last node gets the same budget as prev-to-last node.
     if (i != scores.size() - 1) {
-      // Sign of the score was flipped for sorting, flipping back.
+      // Sign of the score was flipped for sorting, so flip it back.
       const float next_score = -scores[i + 1].first;
       const float q = n->GetQ(-parent_q, search_->kExtraVirtualLoss);
       if (next_score > q) {
         budget_to_spend = std::min(
             budget,
-            int(n->GetP() * factor / (next_score - q) - n->GetNStarted()) + 1);
+            int(n->GetP() * puct_mult / (next_score-q) - n->GetNStarted()) + 1);
```

Comment:
clang-format will add spaces around minus and relayout that line nicely. :)

Reply:
I was hoping that would slip by :P